main.py

```diff
@@ -13,7 +13,7 @@
 from website import settings  # noqa
 from website.app import init_app  # noqa
 
-app = init_app('website.settings', set_backends=True, routes=True)
+application = app = init_app('website.settings', set_backends=True, routes=True)
```

Comment:
This just enables uwsgi compatibility, it looks for something called "application" by default. It shouldn't have any effect on anything else.